Curs/src/style/evaluation.less

```diff
@@ -156,7 +160,7 @@ table {
 }
 
 /* Desktops and laptops ----------- */
-@media screen and (min-width: 1200px) and (max-width: 1024px) {
```

Comment:
Whoops? :stuck_out_tongue:

Reply:
Yeah, I noticed the mistake with min and max. I fixed it. :)